python/kserve/test/test_dataplane.py

```diff
@@ -118,7 +118,8 @@ async def test_model_metadata(self, dataplane_with_model):
 
     async def test_infer(self, dataplane_with_model):
         body = b'{"instances":[[1,2]]}'
-        resp = await dataplane_with_model.infer(self.MODEL_NAME, body)
+        headers = {"content-type": "application/json"}
```

Comment:
This is a slight behavior change, let’s document in the release note that now we require application/json header to decode the json bytes.

Reply:
Updated the release notes